pilopt/src/lib.rs

```diff
@@ -686,6 +886,67 @@ fn constrained_to_constant<T: FieldElement>(
     None
 }
 
+/// Identifies witness columns that are constrained to a (multi)linear expression.
```

Comment:
Can you specify the semantics a bit more concretely? What does it return?

Reply:
Done